source/_components/sensor.history_values.markdown

```diff
@@ -0,0 +1,141 @@
+---
+layout: page
+title: "History Values Sensor"
+description: "Instructions on how to integrate historical values into Home Assistant."
+date: 2018-11-18 14:23
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Utility
+ha_iot_class: "Local Polling"
+ha_release: 0.83
+ha_qa_scale: internal
+---
+
+
+The `history_values` sensor platform provides a way to process and summarize data from another component or platforms, using data from the [history](/components/history/).
+
+It can calculate the minimum, maximum and average values and the range of these values, in a custom time period.
```

Comment:
you talk min and max here... then further ahead it low and peak (confusing)

Reply:
Agreed, will change this to `lowest` and `peak` respectively.

Reply:
Addressed in 56140d2